docs/ar/week03/03.md

```diff
@@ -0,0 +1,27 @@
+| lang-ref | lang | title | translator |
+| ------ | ------ | ------ | ------ |
+| ch.03 | ar | الأسبوع 3 | [aissam out](https://github.com/aissam-out) |

```

Comment:
**This won't work.**
Please, don't change the syntax of the `MD` file.
My script does not understand this format.